### ISSUE TITLE
It seems that 'not-an-idn.com' is an IDN( Internationalized Domain Names), please install 'idn' command first.

### ISSUE THREAD

**Issue by williamdes** [acmesh-official/acme.sh#3324]
`docker run --rm -it neilpang/acme.sh sh`

```
/ # acme.sh --issue --test --standalone          -d "'not-an-idn.com'"
[Sun Dec 27 15:28:53 UTC 2020] It seems that 'not-an-idn.com' is an IDN( Internationalized Domain Names), please install 'idn' command first.
```

I know my domain is invalid but `libidn` `apk add --no-cache --update libidn` is missing from the docker image

**Comment by Neilpang** [comment]
use either  single quotes or double qoutes.
```
-d  'not-an-idn.com'
```

**Comment by williamdes** [comment]
Hey
Please add libidn to the image, this was my request.

**Comment by Neilpang** [comment]
fixed https://github.com/acmesh-official/acme.sh/commit/d6083c68fd3d174fb2ae6025d19965dcf332ac52

please wait a few minutes, and try with the `dev` tag.

Thanks.

**Comment by williamdes** [comment]
> fixed [d6083c6](https://github.com/acmesh-official/acme.sh/commit/d6083c68fd3d174fb2ae6025d19965dcf332ac52)
> 
> please wait a few minutes, and try with the `dev` tag.
> 
> Thanks.

Thank you very much!
I love and recommend acme.sh
